packages/common/src/utils/conditionalFormatting.ts

```diff
@@ -42,28 +42,30 @@ export const getConditionalFormattingConfig = (
 };
 
 export const hasMatchingConditionalRules = (
-    value: number | undefined,
+    value: number | string | undefined,
     config: ConditionalFormattingConfig | undefined,
 ) => {
     if (!config) return undefined;
 
+    const parsedValue = typeof value === 'string' ? Number(value) : value;
```

Comment:
Would the lodash function `toNumber` simplify the code and cover more edge cases?

Reply:
@ZeRego  I added this because metric values were strings 🤷  is that normal?

Reply:
do you think it might be anything else than string or number?

Reply:
metric values should be numbers 🤔

Reply:
Interesting! metric values are strings 🤔 

![Screenshot 2023-01-17 at 15 34 17](https://user-images.githubusercontent.com/9117144/212941430-3283b389-ebb5-4b3b-b73d-e321202f0829.png)